tasks/create-internal-request/create-internal-request.yaml

```diff
@@ -18,20 +18,20 @@ spec:
     - name: request
       type: string
       description: request type
-    - name: params
+    - name: internalRequestParams
       type: string
       description: Internal Request parameters
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the input workspace
```

Comment:
It's a path to a JSON file, not a JSON string. I recently commented on the same thing in a PR of Johnny's so I think you probably copied the description from the same place :)

Reply:
changed.